### PR TITLE
Reformat flip

### PR DESCRIPTION
**Pull request by hnehme99**

### CODE REVIEW

**Comment by hnehme99** [comment]
Close #5847

**Review by Ishticode — CHANGES_REQUESTED** [review]
Hi @hnehme99 
Thank you for the PR 🙂

Could you kindly remove any edits in the `.idea`  and the lint failures taken care of. Let me know if there is something unclear. When done request my review. Thank you

**Comment by hnehme99** [comment]
Hello @Ishticode ! Thank you for your time.

I have done the requested changes but a new lint error, not related to my changes, appeared
(ivy_tests/test_ivy/test_functional/test_extensions.py:167:52: E231 missing whitespace after ',')

should I fix that too? If there is any additional changes please let me know!

File: ivy/container/manipulation.py [hunk 1]
```diff
@@ -451,6 +494,49 @@ def flip(
...
+            an output container having the same data type and
+            shape as ``self`` and whose elements, relative to ``self``, are reordered.
+
+        Examples
+        --------
+        With one :class:`ivy.Container` input:
+
+        >>> x = ivy.Container(a=ivy.array([-1, 0, 1]),
+        ...                   b=ivy.array([2, 3, 4]))
+        >>> y = ivy.flip(x)
```
Comment (Ishticode) [inline ivy/container/manipulation.py:533]: As its a container instance method example, the instance call should be made i.e., `x.flip()`

**Review by Ishticode — CHANGES_REQUESTED** [review]
Hi Looking good. Some minor change as in the comment. Also could you include one example using the `axis` argument for the container/array. Furthermore, could you delete the container, and instance method examples for the docstring in `ivy/functional/ivy/manipulation.py` for `flip`.

**Comment by hnehme99** [comment]
Thank you again @Ishticode ! 

I completed the requested changes. Please let me know if there is any additional changes!

**Review by Ishticode — APPROVED** [review]
Thank you very much @hnehme99 for you contribution and for taking care of flip in torch backend. Not sure how those typehint sneaked in there.

I will get this merged 🙂. Thanks.

**Pull request merged**